release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -3362,6 +3362,59 @@ This section will continue to be updated over time to provide notes on enhanceme
+
+* Previously, for an ingress resource with an `IngressWithoutClassName` alert, the Ingress Controller did not delete the alert when the resource was deleted. The alert continued to open on the {product-title} web console. With this release, the Ingress Controller resets the `openshift_ingress_to_route_controller_ingress_without_class_name` metric to `0` before the controller deletes the ingress resource. The alert is deleted and no longer opens on the web console. (link:https://issues.redhat.com/browse/OCPBUGS-55201[*OCPBUGS-55201*])
+
+* Previously, a race condition between `Go` routines caused the Cluster Version Operator (CVO) to behave inconsistently after the CVO started. With this release, the 'Go' routines synchronization is improved and the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-55156[*OCPBUGS-55156*])
+
+* Previously, {azure-first} network interfaces were created in a `provisioning failed` state, causing potential resource allocation issues. With this release, there is enhanced reliability in {azure-short} network interface controller (NIC) provisioning by adding code to verify and retry the network interface provisioning status by using the {azure-short} software development kit (SDK). (link:https://issues.redhat.com/browse/OCPBUGS-54990[*OCPBUGS-54990*])
+
+* Previously, if a scrape failed, Prometheus incorrectly considered the samples from the next scrape as duplicates and dropped them. This issue affected only the scrape immediately following a failure, while later scrapes were processed correctly. With this release, the scrape following a failure is correctly handled, ensuring that no valid samples are mistakenly dropped. (link:https://issues.redhat.com/browse/OCPBUGS-54942[*OCPBUGS-54942*])
+
+* Previously, during {product-title} software-defined networking (SDN) to `OVNKubernetes` migration with NMState managing `br-ex`, the `mtu-migration`` service that was active before `wait-for-primary-ip` caused a maximum transmission unit (MTU) migration failure. This action resulted in network connectivity issues for the user after migration to {product-version} 4.16. With this release, the services in NMState configurations are reordered, placing 'mtu-migration' after `wait-for-primary-ip`. This arrangement improves stability during migration, and prevents a `mtu-migration` service failure after the first reboot. (link:https://issues.redhat.com/browse/OCPBUGS-54831[*OCPBUGS-54831*])
+
+* Previously, containers that used the SELinux `container_logreader_t` domain to view container logs on a host at `/var/log` could not access logs in the `/var/log/containers` subdirectory. This issue occurred because of a missing symbolic link. With this release, a symbolic link is created for `/var/log/containers` and containers can access the logs in `/var/log/containers`. (link:https://issues.redhat.com/browse/OCPBUGS-54818[OCPBUGS-54818])
```

Comment:
```suggestion
* Previously, containers that used the SELinux `container_logreader_t` domain to view container logs in `/var/log` could not access logs in the `/var/log/containers` subdirectory. This issue occurred because of a missing symbolic link. With this release, a symbolic link is created for `/var/log/containers` and containers can access the logs in `/var/log/containers`. (link:https://issues.redhat.com/browse/OCPBUGS-54818[OCPBUGS-54818])
```